pagespeed/system/system_rewrite_driver_factory.h

```diff
@@ -258,6 +258,18 @@ class SystemRewriteDriverFactory : public RewriteDriverFactory {
     return 1;
   }
 
+  // Set the name of this process, for debugging visibility.
+  virtual void NameProcess(const char* name);
```

Comment:
Right now the factory is where we have process global things, like `InitApr` and `IsServerThreaded`.  I'm adding `NameProcess`, `PrepareForkedProcess`, and `PrepareControllerProcess` which are all pretty small.

We could add a process manager to handle these, but I don't know if it's worth it.  And `PrepareControllerProcess` makes some `SystemRewriteDriverFactory`-internal setup calls.